homework-submission/react-hw4/src/components/stores.js

```diff
@@ -0,0 +1,39 @@
+import { observable, action, computed, configure } from "mobx";
+import Todos from "./todos.json";
+
+configure({ enforceActions: true });
+
+class Stores {
+  @observable
+  todos = Todos;
+
+  @action
+  checker = item => {
+    item.done = !item.done;
+  };
+
+  @action
+  remover = item => {
+    const remove = this.todos.filter(elem => elem !== item);
+    this.todos = remove;
+  };
+
+  @action
+  submitHandler = e => {
+    e.preventDefault();
+    const newTodo = {
+      id: this.todos.length + 1,
```

Comment:
This way of generating ids is error prone, so please use instead the `uuid` npm module or `Math.random()`